README.md

```diff
@@ -24,7 +24,7 @@ These core capabilities are composed to enable private and secure communication
 ```sh
 # Create a TCP Portal Inlet to a Postgres server that is running in
 # a remote private VPC in another cloud.
-ockam tcp-inlet create --from 15432 --to postgres
+ockam tcp-inlet create --from 15432 --via postgres
```

Comment:
Since this snippet doesn't have any context about what commands were executed up to this point, I have a question: is `postgres` a relay or a service?

Reply:
Yes, the name `postgres_relay` might be more realistic.

Reply:
I'll leave the name as it is for now, but I'll keep an eye on it. Thanks!